dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerObjectReport.java

```diff
@@ -63,7 +61,7 @@
     @JsonProperty
     private String uid;
 
-    private Map<TrackerErrorCode, List<TrackerErrorReport>> errorReportsByCode = new HashMap<>();
+    private List<TrackerErrorReport> errorReports = new ArrayList<>();
```

Comment:
I think we can name this field even `errors`. We can also do this later when we rename the `TrackerErrorReport` though 😄